rskj-core/src/main/java/co/rsk/core/bc/TransactionPoolImpl.java

```diff
@@ -463,7 +470,7 @@ private boolean senderCanPayPendingTransactionsAndNewTx(Transaction newTx) {
 
     private Coin getTxBaseCost(Transaction tx) {
         Coin gasCost = tx.getValue();
-        if (bestBlock == null || tx.transactionCost(bestBlock.getNumber(), config.getBlockchainConfig()) > 0) {
+        if (bestBlock == null || tx.transactionCost(constants, activationConfig.forBlock(bestBlock.getNumber())) > 0) {
```

Comment:
I'll just let this mark here to double check this is the right block number for resolving the expected activation configuration